Dockerfile

```diff
@@ -1,18 +1,15 @@
 # Start from a Debian image with the latest version of Go installed
 # and a workspace (GOPATH) configured at /go.
 FROM golang:1.15 AS build
```

Comment:
Yeah any PR to upstream will have to change the values back. It would seem that this is a problem caused by the Go way of including the repo name in the package and module names.

WRT the Dockerfile and Makefile, it may still work without the changes, as the module name should be picked up from the go.mod and not the path, but it is less confusing to have the paths match the module name.